drivers/pinmux/pinmux_mcux_lpc.c

```diff
@@ -12,31 +14,18 @@
 #include <fsl_clock.h>
 #include <fsl_device_registers.h>
 
-#define PORT0_IDX	0u
-#define PORT1_IDX	1u
-
 struct pinmux_mcux_lpc_config {
 	clock_ip_name_t clock_ip_name;
-#ifdef IOPCTL
-	IOPCTL_Type *base;
-#else
-	IOCON_Type *base;
-#endif
-	uint32_t port_no;
+	volatile uint32_t *base;
```

Comment:
To follow the CMSIS standard we could write it as
```
	__IO uint32_t *base;
```
and since we are accessing specifically PIO registers and not all the registers available in the module we could write it as
```
	__IO uint32_t *pio;
```